packages/ruby/lib/readme/payload.rb

```diff
@@ -14,11 +14,11 @@ def initialize(har, info, development:)
       @uuid = UUID.new
     end
 
-    def to_json
+    def to_json(*_args)
       {
         logId: UUID.validate(@log_id) ? @log_id : @uuid.generate,
         group: @user_info,
-        clientIPAddress: "1.1.1.1",
+        clientIPAddress: '1.1.1.1',
```

Comment:
This is probably wrong right? This is how we do it in Node.js:

https://github.com/readmeio/metrics-sdks/blob/684c7452115582d7f29bb102dc534258c55c8ca5/packages/node/src/lib/construct-payload.ts#L132

I think this can be ignored for now, but it will become a problem when we add a Ruby integration test:
https://github.com/readmeio/metrics-sdks/blob/684c7452115582d7f29bb102dc534258c55c8ca5/__tests__/integration-metrics.test.js#L140

Reply:
Yikes yeah this isn't right.

Reply:
Fixed in https://github.com/readmeio/metrics-sdks/pull/537/commits/6101c347f5992f42f88056b41caf86730e54ec3f